modules/lx-autoscale/watchmaker-lx-autoscale.template.cfn.yaml

```diff
@@ -610,6 +620,49 @@ Resources:
                         "log_stream_name": "default_logs_{instance_id}"
                       }
                     }
+            /etc/cfn/scripts/aws-cw-agent/Add-CWL-target.py:
```

Comment:
How about `cloudwatch-applog-config.py` as a script name? Lowercase preferred...

Reply:
updated across lx and win templates